Hevelyn (a3b5 b1+) (#3)
BrunoDamacena/wca-regulations-translations: portuguese-brazilian/wca-regulations.md

        - A3b2) O competidor inicia a tentativa confirmando que está pronto, e então o juiz descobre o quebra-cabeça e começa a marcar o tempo de inspeção.
        - A3b3) Quando 8 segundos de inspeção tiverem passado, o juíz diz "8 SEGUNDOS".
        - A3b4) Quando 12 segundos de inspeção tiverem passado, o juíz diz "12 SEGUNDOS".

>         - A3b5)O juiz para a cronometragem do tempo de inspeção assim que o competidor levanta as mãos do timer, iniciando-o. (Ver [Regra A4d](regulations:regulation:A4d)).

Reviewer: ```suggestion
        - A3b5) O juiz para a cronometragem do tempo de inspeção assim que o competidor levanta as mãos do timer, iniciando-o. (Ver [Regra A4d](regulations:regulation:A4d)).
```

    - A3c) O competidor pode pegar o quebra-cabeça durante a inspeção.
        - A3c1) O competidor não deve aplicar movimentos ou mudanças no alinhamento intencionais durante a inspeção. Pena: desqualificação da tentativa (DNF). Exceção: ver [Regra: A3c2](regulations:regulation:A3c2) e [Regra A3c5](regulations:regulation:A3c5).
        - A3c2) Se as peças do quebra-cabeça não estiverem totalmente alinhadas quando o pote for removido, o competidor pode alinhar as faces, contanto que os desalinhamentos estejam dentro dos limites da [Regra 10f](regulations:regulation:10f).
        - A3c3) O competidor pode zerar o timer antes de começar a resolver.
        - A3c4) Para o Square-1, se o uso de um objeto fino foi determinado pela equipe de organização (ver [Regra A2b1](regulations:regulation:A2b1)), o competidor pode remover o objeto do quebra-cabeça durante a inspeção.

>         - A3c5) Para o Square-1, o competidor não deve ser penalizado por qualquer movimento acidental dentro dos limites da [Regra 10f](regulations:regulations:10f).

Reviewer: ```suggestion
        - A3c5) Para o Square-1, é recomendável que o competidor não seja penalizado por qualquer movimento acidental dentro dos limites da [Regra 10f](regulations:regulations:10f).
```

    - A3d) No final da inspeção, o competidor coloca o quebra-cabeça sobre o tapete, em qualquer orientação. O quebra-cabeça deve estar inteiramente em cima do tapete e não deve estar inteiramente ou parcialmente sobre o Stackmat timer. Pena: pena de tempo (+2 segundos).
- A4) Iniciando a resolução:

>     - A4b) O competidor usa seus dedos para tocar a parte elevada do timer Stackmat. Suas palmas têm que estar para baixo, e posicionadas no lado do timer mais próximo dele. Pena: pena de tempo (+2 segundos).

Reviewer: ```suggestion
    - A4b) O competidor usa seus dedos para tocar a parte elevada do timer Stackmat. Suas palmas devem estar para baixo, e posicionadas no lado do timer mais próximo dele. Pena: pena de tempo (+2 segundos).
```

>         - A4b1) O competidor não pode ter contato físico com o quebra-cabeça enquanto estiver começando a resolução. Pena: pena de tempo (+2 segundos).

Reviewer: ```suggestion
        - A4b1) O competidor não deve ter contato físico com o quebra-cabeça enquanto estiver começando a resolução. Pena: pena de tempo (+2 segundos).
```

    - A4d) Se o timer Stackmat estiver sendo usado, é recomendável que o competidor mantenha suas mãos no timer até que ele veja uma luz verde no timer. Ele inicia a resolução removendo sua(s) mão(s) do timer. O competidor começa a resolução quando o timer for iniciado.
        - A4d1) O competidor deve iniciar a resolução dentro de 15 segundos após o início da inspeção. Pena: pena de tempo (+2 segundos).
        - A4d2) O competidor deve iniciar a resolução dentro de 17 segundos após o início da inspeção. Pena: desqualificação da tentativa (DNF).
        - A4d3) Se um cronômetro estiver sendo usado, o juiz inicia o cronômetro no momento em que o competidor iniciar a resolução.
    - A4e) Pena de tempo para o início da resolução são cumulativas.
- A5) Durante a resolução:

>     - A5a) Enquanto estiver inspecionando ou resolvendo o quebra-cabeça, o competidor não pode se comunicar com ninguém além do juiz ou Delegado da WCA. Pena: desqualificação da tentativa (DNF). Exceção: Se o competidor não ganhar nenhuma vantagem por se comunicar com qualquer outra pessoa além do juiz ou do Delegado da WCA, o resultado pode permanecer, a critério do Delegado da WCA.

Reviewer: ```suggestion
    - A5a) Enquanto estiver inspecionando ou resolvendo o quebra-cabeça, o competidor não deve se comunicar com ninguém além do juiz ou Delegado da WCA. Pena: desqualificação da tentativa (DNF). Exceção: Se o competidor não ganhar nenhuma vantagem por se comunicar com qualquer outra pessoa além do juiz ou do Delegado da WCA, o resultado pode permanecer, a critério do Delegado da WCA.
```

    - A5b) Enquanto estiver inspecionando ou resolvendo o quebra-cabeça, o competidor não deve receber assistência de ninguém ou de nenhum objeto além da superfície (ver também [Regra 2i](regulations:regulation:2i)). Pena: desqualificação da tentativa (DNF).
    - A5c) O competidor pode segurar o quebra-cabeça contra a superfície para ajudá-lo a operá-lo (ver [Regra 71fd](regulations:regulation:71fd)).
- A6) Finalizando a resolução:
    - A6a) O competidor solta o quebra-cabeça antes de parar a resolução, e para a resolução parando o timer. Se um cronômetro estiver sendo usado, o juiz para o cronômetro assim que o competidor fizer isso.

>         - A6a1) Se o juíz não estiver imediatamente disponível para parar o cronômetro quando dado cronômetro estiver em uso, o competidor pode pará-lo.

Reviewer: ```suggestion
        - A6a1) Se o juiz não estiver imediatamente disponível para parar o cronômetro quando o cronômetro estiver em uso, o competidor pode pará-lo.
```

    - A6b) O competidor é responsável por parar o timer Stackmat corretamente.
        - A6b1) Se o timer parar antes do competidor ter resolvido o quebra-cabeça e mostrar um tempo abaixo de 0,06 segundos, a tentativa é substituída por uma tentativa adicional. Exceção: Nenhuma tentativa extra será necessária se o Delegado da WCA determinar que o cronômetro foi parado deliberadamente ou como resultado de interação com o cronômetro de forma incomum (p. ex., fazendo apenas contato mínimo com o cronômetro, não levantando as mãos do cronômetro de forma limpa).
        - A6b2) Se o timer parar antes do competidor ter resolvido o quebra-cabeça e mostrar um tempo de 0,06 segundos ou maior, a tentativa é desqualificada (DNF). Exceção: o Delegado da WCA pode dar uma tentativa extra para o competidor se acreditar que o cronômetro pode ter apresentado defeito e que o competidor não o parou deliberadamente.

>     - A6c) O competidor tem que soltar totalmente o quebra-cabeça antes de parar o timer. Pena: desqualificação da tentativa (DNF). Exceção: Se nenhum movimento ou qualquer mudança no alinhamento tiverem sido aplicados após parar o timer e antes de soltar o quebra-cabeça, poderá ser aplicada uma pena de tempo (+2 segundos), a critério do juiz.

Reviewer: ```suggestion
    - A6c) O competidor deve soltar totalmente o quebra-cabeça antes de parar o timer. Pena: desqualificação da tentativa (DNF). Exceção: Se nenhum movimento ou qualquer mudança no alinhamento tiverem sido aplicados após parar o timer e antes de soltar o quebra-cabeça, poderá ser aplicada uma pena de tempo (+2 segundos), a critério do juiz.
```

>     - A6d) O competidor tem que parar o timer Stackmat usando ambas as mãos, colocadas planas sobre os sensores e com as palmas para baixo. Pena: pena de tempo (+2 segundos).

Reviewer: ```suggestion
    - A6d) O competidor deve parar o timer Stackmat usando ambas as mãos, colocadas planas sobre os sensores e com as palmas para baixo. Pena: pena de tempo (+2 segundos).
```

>     - A6e) Depois de soltar o quebra-cabeça, o competidor não pode tocar, aplicar um movimento, ou qualquer mudança no alinhamento do quebra-cabeça até que o juiz o tenha inspecionado.

Reviewer: ```suggestion
    - A6e) Depois de soltar o quebra-cabeça, o competidor não deve tocar, aplicar um movimento, ou qualquer mudança no alinhamento do quebra-cabeça até que o juiz o tenha inspecionado.
```

        - A6e1) Penalidade para aplicar um movimento ao quebra-cabeça após parar o timer: desqualificação da tentativa (DNF).
        - A6e2) Penalidade para tocar o quebra-cabeça após parar o timer sem aplicar nenhum movimento: pena de tempo (+2 segundos). Exceção: Se, imediatamente após tocar o quebra-cabeça, o competidor brevemente tocar o quebra-cabeça sem afetar o seu estado, a pena de tempo deve ser desconsiderada, a critério do juiz (ver [Regra 11f](regulations:regulation:11f)).

>     - A6f) O competidor não deve zerar o timer até que o ambos competidor juíz tenham assinado a súmula. Pena: desqualificação da tentativa (DNF), a critério do juiz. Exceção: ver [Regra A7c5](regulations:regulation:A7c5).

Reviewer: ```suggestion
    - A6f) O competidor não deve zerar o timer até que ambos competidor e juíz tenham assinado a súmula. Pena: desqualificação da tentativa (DNF), a critério do juiz. Exceção: ver [Regra A7c5](regulations:regulation:A7c5).
```

>         - A6f1) Se o competidor zerar o timer antes que o resultado tenha sido registrado completamente, o juiz não pode escrever o resultado a partir de sua memória ou de evidência fotográfica, e tem que desqualificar a tentativa (DNF).

Reviewer: ```suggestion
        - A6f1) Se o competidor zerar o timer antes que o resultado tenha sido registrado completamente, o juiz não deve escrever o resultado a partir de sua memória ou de evidência fotográfica, e deve desqualificar a tentativa (DNF).
```

>     - A6g) O juiz determina se o quebra-cabeça está resolvido. Se o quebra-cabeça estiver desalinhado, este não pode ser tocado antes que o juiz tenha determinado o seu estado final.

Reviewer: ```suggestion
    - A6g) O juiz determina se o quebra-cabeça está resolvido. Se o quebra-cabeça estiver desalinhado, este não deve ser tocado antes que o juiz tenha determinado o seu estado final.
```

>     - A6h) Em caso de disputa, nenhum movimento ou alinhamento pode ser aplicado ao quebra-cabeça até que a disputa seja resolvida.

Reviewer: ```suggestion
    - A6h) Em caso de disputa, nenhum movimento ou alinhamento deve ser aplicado ao quebra-cabeça até que a disputa seja resolvida.
```

    - A6i) Penas de tempo para a parte de terminar a resolução são cumulativas.
- A7) Registrando resultados:
    - A7a) O juiz diz ao competidor o resultado.

>         - A7a1) O juiz deve indicar ao competido que o quebra-cabeça está resolvido, ou que uma penalidade será aplicada.

Reviewer: ```suggestion
        - A7a1) É recomendável que o juiz indique ao competidor que o quebra-cabeça está resolvido, ou que uma penalidade será aplicada.
```

>     - A7b) O juiz anota o resultado em uma súmula e assina ao lado do resultado para atestar que o resultado está correto, completo, formatado corretamente e legível. É recomendado que a assinatura do juiz seja consistente durante toda a competição e que, através dela, seja possível identificar o juiz.

Reviewer: ```suggestion
    - A7b) O juiz anota o resultado em uma súmula e assina ao lado do resultado para atestar que o resultado está correto, completo, formatado corretamente e legível. É recomendável que a assinatura do juiz seja consistente durante toda a competição e que, através dela, seja possível identificar o juiz.
```

>         - A7b1) Se alguma pena for aplicada, o juiz anota o resultado original mostrado no timer, juntamente com quaisquer penas. O formato deve ser "X + T + Y = F", em que X representa a soma das penas de tempo antes/no início da resolução, T representa o tempo mostrado no timer, Y representa a soma das  de tempo durante/depois da resolução, e F representa o resultado final (p. ex. 2 + 17,65 + 2 = 21,65). Se X e/ou Y forem 0, os termos 0 são omitidos (p. ex. 17,65 + 2 = 19,65).

Reviewer: ```suggestion
        - A7b1) Se alguma pena for aplicada, o juiz anota o resultado original mostrado no timer, juntamente com quaisquer penas. É recomendável que formato seja "X + T + Y = F", em que X representa a soma das penas de tempo antes/no início da resolução, T representa o tempo mostrado no timer, Y representa a soma das  de tempo durante/depois da resolução e F representa o resultado final (p. ex. 2 + 17,65 + 2 = 21,65). Se X e/ou Y forem 0, os termos 0 são omitidos (p. ex. 17,65 + 2 = 19,65).
```

>     - A7c) O competidor precisa verificar o resultado registrado e assinar (ou colocar sua inicial) a súmula para reconhecer que o resultado está correto, completo, devidamente formatado e claramente legível. Isso conclui a tentativa.

Reviewer: ```suggestion
    - A7c) O competidor deve verificar o resultado registrado e assinar (ou colocar sua inicial) a súmula para reconhecer que o resultado está correto, completo, devidamente formatado e claramente legível. Isso conclui a tentativa.
```

>         - A7c1) Se o competidor ou o juiz se recusar a aceitar e assinar a súmula, o Delegado da WCA tem que resolver a disputa.

Reviewer: ```suggestion
        - A7c1) Se o competidor ou o juiz se recusar a aceitar e assinar a súmula, o Delegado da WCA deve resolver a disputa.
```

        - A7c2) Se o competidor assinar intencionalmente (ou marcar de outra forma) uma tentativa antes que a tenha iniciado, ele perde o direito àquela tentativa e o resultado será considerado DNS. Exceção: para o 3x3x3 Multi-Vendado, o competidor pode escrever na súmula o número de cubos de sua tentativa.
        - A7c3) Se o competidor assinar (ou marcar de outra forma) uma tentativa depois que já a tenha iniciado mas antes que o juiz termine de registrar o resultado, a tentativa será considerada não concluída (DNF).

>         - A7c4) O competidor não deve assinar uma tentativa antes que o juiz tenha registrado o resultado e assinado. O competidor é responsável em última instância por garantir que ele e o juiz assinem a tentativa. Se uma tentativa tiver uma ou as duas assinaturas faltando após o juiz entregar a súmula ao anotador de resultados (ver [Regra A7f](regulations:regulations:A7f)), esta será considerada não concluída (DNF).

Reviewer: ```suggestion
        - A7c4) É recomendável que o competidor não assine uma tentativa antes que o juiz tenha registrado o resultado e assinado. O competidor é responsável em última instância por garantir que ele e o juiz assinem a tentativa. Se uma tentativa tiver uma ou as duas assinaturas faltando após o juiz entregar a súmula ao anotador de resultados (ver [Regra A7f](regulations:regulations:A7f)), esta será considerada não concluída (DNF).
```

        - A7c5) Assinaturas ausentes encontradas antes do fim da competição (ver [Regra 9u](regulations:regulation:9u)) podem ser adicionadas pelo juiz e/ou competidor, a critério do Delegado da WCA.

>         - A7c6) É recomendado que assinaturas de competidores sejam consistentes durante toda a competição e, através dela, seja possível identificar o competidor. Competidores não devem assinar incorretamente de maneira intencional. Pena: desqualificação da tentativa (DNF), a critério do Delegado da WCA.

Reviewer: ```suggestion
        - A7c6) É recomendável que assinaturas de competidores sejam consistentes durante toda a competição e, através dela, seja possível identificar o competidor. Competidores não devem assinar incorretamente de maneira intencional. Pena: desqualificação da tentativa (DNF), a critério do Delegado da WCA.
```

        - A7c7) Não é recomendável que o competidor escreva na súmula em nenhum outro lugar além da área destinada para sua assinatura.

>         - A7c8) Se uma assinatura for esquecida devido a inexperiência do competidor, ela pode ser adicionada a qualquer momento, a critério do Delegado da WCA.Se não for mais possível que a súmula seja assinada fisicamente pelo competidor ou pelo juiz, uma declaração de que o resultado foi registrado corretamente pode ser aceita como assinatura, a critério do Delegado da WCA.

Reviewer: ```suggestion
        - A7c8) Se uma assinatura for esquecida devido a inexperiência do competidor, ela pode ser adicionada a qualquer momento, a critério do Delegado da WCA. Se não for mais possível que a súmula seja assinada fisicamente pelo competidor ou pelo juiz, uma declaração de que o resultado foi registrado corretamente pode ser aceita como assinatura, a critério do Delegado da WCA.
```
